packages/orchestration/src/exos/local-chain-facade.js

```diff
@@ -36,17 +37,22 @@ const prepareLocalChainFacadeKit = (
   {
     makeLocalOrchestrationAccountKit,
     localchain,
-    storageNode,
+    storageNode, // consider making an accounts childNode TODO #9066
```

Comment:
TODO goes at the front, and is meant to be done. If you're not sure the change should be made, make the TODO to consider it. I.e. `TODO consider`. Also in general TODO and other ALLCAPS directives go at the start